src/sentry/api/endpoints/organization_index.py

```diff
@@ -28,17 +28,22 @@
 )
 from sentry.services.organization.provisioning import organization_provisioning_service
 from sentry.signals import org_setup_complete, terms_accepted
+from sentry.types.region import get_local_region
 
 
 class OrganizationPostSerializer(BaseOrganizationSerializer):
     defaultTeam = serializers.BooleanField(required=False)
     agreeTerms = serializers.BooleanField(required=True)
+    region = serializers.CharField(required=True)
```

Comment:
I think making this required will break things in between the backend change rolling out and users refreshing to load in the UI changes which start sending `region` - you may need to do this in two steps starting with a non-required value and then making it required later

Reply:
This should only break for the test-region to start. I've remove the validator below when the multi-region selector is disabled.

Reply:
I ended up just removing these API changes entirely in favor of a form refactor